app/dao/services_dao.py

```diff
@@ -366,127 +375,188 @@ def dao_remove_user_from_service(service, user):
 
 def delete_service_and_all_associated_db_objects(service):
 
-    def _delete_commit(query):
-        query.delete(synchronize_session=False)
+    def _delete_commit(stmt):
+        db.session.execute(stmt, execution_options={"synchronize_session": False})
```

Comment:
This is [legacy](https://docs.sqlalchemy.org/en/20/orm/queryguide/query.html#sqlalchemy.orm.Query.execution_options).  "execution_options" should be part of the query statement.  See the 2.0 example in inbound_sms_dao.py.

Reply:
Team agreed. No changes required. Attaching to exec instead of stmt.